### PR TITLE
OCPBUGSM-23927 - correct openshift version recieved for day2 cluster …

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**
…to be of the format used by version handler

### CODE REVIEW

**Review by YuviGold — APPROVED** [review]
LGTM

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -715,14 +715,20 @@ func (b *bareMetalInventory) RegisterAddHostsCluster(ctx context.Context, params
...
-	log.Infof("Register add-hosts-cluster: %s with id %s", clusterName, id.String())
+	log.Infof("Register add-hosts-cluster: %s, id %s, version %s", clusterName, id.String(), inputOpenshiftVersion)
 
 	if cluster.ClusterExists(b.db, *id) {
 		return common.NewApiError(http.StatusBadRequest, fmt.Errorf("AddHostsCluster for AI cluster %s already exists", id))
 	}
 
+	openshiftVersion, err := b.versionsHandler.GetSupportedVersionFormat(inputOpenshiftVersion)
+	if err != nil {
+		log.WithError(err).Errorf("Failed to get opnshift version supported by versions map from version %s", inputOpenshiftVersion)
```
Comment (romfreiman) [inline internal/bminventory/inventory.go:728]: OpenShift

File: internal/bminventory/inventory.go [hunk 2]
```diff
@@ -715,14 +715,20 @@ func (b *bareMetalInventory) RegisterAddHostsCluster(ctx context.Context, params
...
+		return common.NewApiError(http.StatusBadRequest, fmt.Errorf("Failed to get opnshift version supported by versions map from version %s", inputOpenshiftVersion))
```
Comment (romfreiman) [inline internal/bminventory/inventory.go:729]: OpenShift

**Comment by romfreiman** [comment]
https://github.com/openshift/machine-config-operator/blob/release-4.7/docs/OSUpgrades.md

**Pull request merged**